app/controllers/concerns/authenticated_state_file_intake_concern.rb

```diff
@@ -5,13 +5,26 @@ module AuthenticatedStateFileIntakeConcern
     before_action :require_state_file_intake_login
   end
 
+  def current_intake
+    # NOTE: This intake is different from the unauthenticated value, stored in the regular session
+    # (Which the unauthenticated sessions controller uses)
+    unless @intake
+      current_intake_name = "current_#{question_navigator.intake_class.name.underscore}"
+      @intake = self.send(current_intake_name)
+    end
+    @intake
+  end
+
   private
 
   def card_postscript
     I18n.t("state_file.state_file_pages.card_postscript.responses_saved_html").html_safe
   end
 
```

Comment:
Your idea about having an explicit authenticated intake is valuable and could even be used to improve and refactor this file a little. It was something I suggested when we did the earlier auth work, but was seen a "nice to have" refactor that didn't get done.

```suggestion
def current_authenticated_intake
    case params[:us_state]
    when "ny" then current_state_file_ny_intake
    when "az" then current_state_file_az_intake
end
```